app/lib/server/methods/filterBadWords.js

```diff
@@ -16,13 +16,13 @@ callbacks.add('beforeSaveMessage', function(message) {
 			};
 		}
 		const filter = new Filter(options);
-
+		console.log('ed1');
 		if (whiteList?.length) {
 			whiteList = whiteList.split(',').map((word) => word.trim());
 			filter.removeWords(...whiteList);
 		}
 
-		message.msg = filter.clean(message.msg);
+		message.msg ? message.msg = filter.clean(message.msg) : message.title = filter.clean(message.title);
```

Comment:
I think it'd be better to do something like
```

Reply:
In this statement I wanted to make sure that if msg exists, we assign msg = filter.clean(message.msg). Whereas if it did not exist, we assign title = filter.clean(message.title)

Reply:
you're right, i misread it